teps/0115-tekton-catalog-reorganization.md

```diff
@@ -0,0 +1,284 @@
+---
+status: implementable
+title: Tekton Catalog Reorganization
+creation-date: '2022-07-12'
+last-updated: '2022-07-12'
+authors:
+- "@jerop"
+- "@vdemeester"
+- "@vinamra28"
+- "@QuanZhang-William"
```

Comment:
@vdemeester @vinamra28 @QuanZhang-William added you as co-authors given prior discussions about this work - please let me know if you'd prefer that I can change this 😀